VL.ImGui.Stride/src/ImGuiRenderer.FontTexture.cs

```diff
@@ -0,0 +1,32 @@
+﻿using ImGuiNET;
+using Stride.Graphics;
+
+namespace VL.ImGui
+{
+    partial class ImGuiRenderer
+    {
+        void BuildImFontAtlas(ImFontAtlasPtr atlas, float scaling)
+        {
+            atlas.BuildImFontAtlas(scaling, _context, fonts);
+
+            unsafe
+            {
+                atlas.GetTexDataAsRGBA32(out byte* pixelData, out int width, out int height, out int bytesPerPixel);
+                if (width == 0 || height == 0)
+                {
+                    // Something went wrong, load default font
+                    atlas.Clear();
+                    _context.Fonts.Clear();
+                    atlas.AddFontDefault();
+                    atlas.GetTexDataAsRGBA32(out pixelData, out width, out height, out bytesPerPixel);
+                }
+
+                var newFontTexture = Texture.New2D(device, width, height, PixelFormat.R8G8B8A8_UNorm_SRgb, TextureFlags.ShaderResource);
```

Comment:
Whether Srgb or not should be dependent on the ColorSpace property of the device. There're easy helper methods for this I remember.